examples/chef/nrfconnect/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ if (CONFIG_CHIP_LIB_SHELL)
 endif()
 
 target_sources(app PRIVATE
+    ${CHEF}/common/DataModelUtils.cpp
```

Comment:
This will be needed for other platforms as well. Did you search for `chef-air-quality.cpp` elsewhere to see if any other builds (e.g. ESP32) also need it?